src/d3a/models/area/event_dispatcher.py

```diff
@@ -205,3 +209,102 @@ def _delete_past_agents(self, area_agent_member):
                     agent.higher_market = None
                     agent.lower_market = None
                 del area_agent_member[pm]
+
+
+class RedisAreaDispatcher(AreaDispatcher):
+    def __init__(self, area, redis_comm):
+        super().__init__(area)
+        self.redis_comm = redis_comm
```

Comment:
You can rename redis_comm to just redis IMO, this will make the code much more concise and equally readable.